ci/docker/tizen/tools/cache-checksum.sh

```diff
@@ -40,7 +40,7 @@ if [[ "$COMMAND" == "restore" ]]; then
     fi
 
     # Set mtime of files in $TARGET_DIRS to OLD time.
-    for d in $(find $TARGET_DIRS -type d -not -path "*/.git*"); do
+    for d in $(find $TARGET_DIRS -type d); do
         touch -c -m -d @1600000000 $d/* &
```

Comment:
I'm just curious, is there a specific meaning to the @1600000000 time or is it just a random time in the past?

Reply:
> I'm just curious, is there a specific meaning to the @1600000000 time or is it just a random time in the past?

This is an arbitrary time value. In fact, any value before the time the cache was created is fine.